public/app/plugins/panel/dashlist/DashList.tsx

```diff
@@ -37,9 +37,11 @@ async function fetchDashboards(options: DashListOptions) {
   if (options.showSearch) {
     const params = {
       limit: options.maxItems,
-      query: options.query,
+      query: /^\$.+/.test(options.query) ? replaceVars(options.query, {}, 'text') : options.query,
```

Comment:
always pass it through replaceVars, there are multiple syntaxes so testing if the string contains variables is not needed

Reply:
This is done.